src/services/configServices/SettingsService.js

```diff
@@ -140,12 +140,15 @@ class SettingsService {
     const clonedCurrentData = _.cloneDeep(currentData)
     Object.keys(updatedData).forEach((field) => {
       if (field === "social_media") {
+        // Set social_media field if it doesn't exist
+        if (!clonedCurrentData[field]) clonedCurrentData[field] = {}
```

Comment:
Nit: Can we add curly braces around this one line if condition for maintainability ya

Reply:
Sure, added in https://github.com/isomerpages/isomercms-backend/pull/496/commits/7e4c001c0336c36010f40ec427e0d030b007558a